jabichris/Politico-v1
# #163934899 Create a user

#### What does this PR do?
This PR will help me in getting the tables so that I can start to add controllers to consume my database.

#### Description of Task to be completed?
I want to get tables where I will be storing different data from my methods

#### How should this be manually tested?
After this branch being merged on develop and you can clone this link [git@github.com](mailto:git@github.com):jabichris/Politico-v1.git . then you should run npm install for installing all dependencies and this script (npm run db) for the creation of database tables.

#### What are the relevant pivotal tracker stories?
#163934899
#### Screenshots (if appropriate)
![screen shot 2019-02-19 at 19 04 30](https://user-images.githubusercontent.com/33568373/53033230-43853000-3479-11e9-9c1b-b9c5bbced067.png)
